composer.json

```diff
@@ -115,7 +115,7 @@
         ],
         "stan-tests": "phpstan.phar analyze -c tests/phpstan.neon",
         "stan-baseline": "phpstan.phar --generate-baseline",
-        "stan-setup": "cp composer.json composer.backup && composer require --dev phpstan/phpstan:~1.9.0 psalm/phar:~5.1.0 && mv composer.backup composer.json",
+        "stan-setup": "cp composer.json composer.backup && composer require --dev phpstan/phpstan:^1.9 psalm/phar:^5.4 && mv composer.backup composer.json",
```

Comment:
The reason we stopped doing this was constant destructive changes in releases from phpstan/psalm. We'd constantly have a broken build.

Reply:
Then we have to change our current CI config to use stan-setup instead of always installing the latest version.

Reply:
If it's more stable now, we could switch back to this. Up to @ADmad

Reply:
Using the same stan-setup sounds good to me
The advantage would be that master is always green, and one can manually update the bounds every x weeks.

Reply:
Since we are sharing the GA CI config accross our 5.x plugins as well this means, that we have to define/maintain all occurencies of `stan-setup` per repository if we go this way.
But I am fine with that so we don't get into the same situation of randomly broken builds again as @othercorey already mentioned.

Reply:
Sounds reasonable 
This allows each repo to update individually